### DISCUSSION TITLE
Расширение возможностей рынка

### DISCUSSION THREAD

**Discussion by sobakka265** [octothorp-team/public-dobrograd#803] [Предложения]
Смотря на действующий рынок я задаюсь вопросом,почему нельзя продавать/покупать готовую еду(взамен на удалённый автомат),почему нету чёрного рынка или его подобия? Я предлагаю добавить новые разделы,раздел еды-для обычных граждан или иных лиц.А вот насчёт чёрного рынка я думаю так,доступ к нему должны иметь только токари,старшие токари,фармацевты и провизоры(ясно дело что продавать они могут только оружие,наркоту и т.д,дабы обычные аптеки работали исправно) доступен он будет только выше перечисленным и членам офиц.банд

**Comment by Wani4ka** [comment]
Рыбу же вроде можно продавать

**Reply by sobakka265** [reply]
да,не замечал их среди обилия иного,да и расположены они как то сумбурно
